Say Their Names/Source/Controller/Home/HomeView.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  HomeView.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class HomeView: UIView {
```

Comment:
If you are not going to subclass it, then it is final.

```suggestion
final class HomeView: UIView {
```

Reply:
done